test/diagnostics/warnings_as_errors_rules.swift

```diff
@@ -1,10 +1,7 @@
 // RUN: not %target-swift-frontend -typecheck -diagnostic-style llvm -warnings-as-errors %s 2>&1 | %FileCheck %s --check-prefix=CHECK-WAE
-// RUN: not %target-swift-frontend -typecheck -diagnostic-style llvm -Werror availability_deprecated %s 2>&1 | %FileCheck %s --check-prefix=CHECK-WE-GROUP
-// RUN: not %target-swift-frontend -typecheck -diagnostic-style llvm -Werror deprecated %s 2>&1 | %FileCheck %s --check-prefix=CHECK-WE-SUPERGROUP
```

Comment:
I'm removing `-SUPERGROUP` tests here because we don't currently have any. I'll bring it back in a future PR (probably with the `Concurrency` supergroup).

Reply:
Yeah, that's fine.